Cargo.toml

```diff
@@ -27,6 +27,7 @@ serde-wasm-bindgen = { version = "0.3.0", optional = true }
 swc_common = "0.11.6"
 swc_ecmascript = { version = "0.57.0", features = ["dep_graph", "parser"] }
 termcolor = "1.1.2"
+text_lines = "0.1.2"
```

Comment:
This is the crate I recently created https://docs.rs/text_lines/0.1.2/text_lines/struct.TextLines.html

It's also used in some of Deno's dependencies now.